libs/product/state/src/reducers/product/product-reducer-state.interface.ts

```diff
@@ -5,7 +5,7 @@ import { DaffProduct } from '@daffodil/product';
  * Interface for product state.
  */
 export interface DaffProductReducerState {
-  selectedProductId: DaffProduct['id'];
+  selectedProductId: DaffProduct['id'] | DaffProduct['url'];
```

Comment:
This seems like it might become problematic. But I think this field is deprecated, so maybe just revert this change and we'll just remove this field entirely in a future PR.

Reply:
removed